resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -60,6 +61,7 @@
     private boolean writableStackTraceEnabled = DEFAULT_WRITABLE_STACK_TRACE_ENABLED;
```

Comment:
We could remove `waitDurationInOpenState`  and replace it by a constant waitIntervalFunction